horovod/spark/keras/estimator.py

```diff
@@ -140,6 +140,9 @@ class KerasEstimator(HorovodEstimator, KerasEstimatorParamsReadable,
         train_steps_per_epoch: Number of steps to train each epoch. Useful for testing that model trains successfully.
                                Defaults to training the entire dataset each epoch.
         validation_steps_per_epoch: Number of validation steps to perform each epoch.
+        transformation_fn: Optional custom function to execute before passing the loaded data into the train and validation steps.
```

Comment:
Here we should explain how this works a little more.  A reference to the Petastorm documentation would be useful.  Something like: `Optional function that takes a row as its parameter and returns a modified row that is then fed into the train or validation step.  This transformation is applied after batching.  See Petastorm [TransformSpec](link) for more details`.